build/Program.cs

```diff
@@ -30,22 +30,28 @@ public static async Task Main(string[] args)
                 DependsOn(Build),
                 () => RunAsync("dotnet", $"test tests/PseudoLocalizer.Tests/PseudoLocalizer.Tests.csproj -c Release --no-build --verbosity=normal"));
 
-            if (IsTravis() && IsMasterBranch())
-            {
-                Target(
-                    Pack,
-                    DependsOn(Test),
-                    () => RunAsync("dotnet", $"pack src/PseudoLocalizer/PseudoLocalizer.csproj -c Release -o ../../{ArtifactsDir} --no-build"));
+            Target(
+                Pack,
+                DependsOn(Build),
+                () => RunAsync("dotnet", $"pack src/PseudoLocalizer/PseudoLocalizer.csproj -c Release -o ../../{ArtifactsDir} --no-build"));
+
+            Target(
+                Publish,
+                DependsOn(Pack),
+                async () =>
+                {
+                    if (IsTravis() && IsMasterBranch())
```

Comment:
Side note: `IsTravis()` seems redundant, since `IsMasterBranch()` already relies on env vars that are only in Travis. I think `IsTravis()` could be removed, and `IsMasterBranch()` renamed to `IsMasterBranchInTravis()`.

Reply:
You're right. I'll change it.